documentation/source/library-reference/common-dylan/byte-vector.rst

```diff
@@ -14,6 +15,8 @@ The byte-vector Module
 
      - :type:`<byte>`
 
+.. <byte> is defined in the dylan library but documented here (for reasons?).
```

Comment:
It's defined in the dylan library because it's referenced by language extensions and implementation details that have to be there.

Reply:
Right, I was a bit lazy with that comment and I'll remove it.

I was basically wondering why we don't document it in `dylan-extensions` directly instead of somewhere that it is re-exported from.